task/orka-deploy/0.2/samples/use-ssh-key.yaml

```diff
@@ -0,0 +1,40 @@
+# ###
+# This example demonstrates how to use an SSH key to connect to the Orka VM.
+# You will first need to copy the public key to the VM and commit or save an
+# image using `orka image commit` or `orka image save` and specify that base image.
+# in the TaskRun or Pipeline as a param.
+#
+# You must specify the Task param ssh-key="true" in order to use an SSH key.
+#
+# ###
+# You can create a Kubernetes secret with the SSH credentials as follows:
+# kubectl create secret generic orka-ssh-key --from-file=id_rsa=/path/to/id_rsa --from-literal=username=<username>
+# ###
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: use-ssh-key
+spec:
+  taskRef:
+    name: orka-full
```

Comment:
Unless there is a reason not to, I think we should keep this example as a `PipelineRun`. The `orka-deploy` task is not used here at all, and a big part of the purpose of the example was to illustrate how to use the init / deploy / teardown tasks together